dvc/repo/live.py

```diff
@@ -14,27 +9,15 @@
     from dvc.output import Output
     from dvc.repo import Repo
 
-webbrowser_open = once_per_args(webbrowser.open)
```

Comment:
Might be easy to just use `autouse=` mocked object than the rollback. Let me see.

Reply:
@skshetry Let's continue in https://github.com/iterative/dvc/pull/7032